website/src/index-page.tsx

```diff
@@ -498,10 +502,31 @@ function ButtonLink({
 function TextLink({
   children,
   className,
+  noAnchor,
+  href,
   ...props
-}: ComponentProps<typeof Anchor>) {
+}: ComponentProps<typeof Anchor> & { noAnchor?: boolean }) {
+  const router = useRouter()
+  if (noAnchor) {
+    return (
+      <span
+        onClick={(e) => {
+          e.preventDefault()
+          router.push(href)
+        }}
+        className={clsx('text-primary-500', className)}
+        {...props}
+      >
+        {children}
+      </span>
```

Comment:
we can use button here so link will be focusable
```suggestion
      <button
        onClick={(e) => {
          e.preventDefault()
          router.push(href)
        }}
        className={clsx('text-primary-500', className)}
        {...props}
      >
        {children}
      </button>
```

Reply:
Good thinking, made the change https://github.com/dotansimha/graphql-yoga/pull/1940/commits/6c1f7a428521a3ed64ede172bebf335757db353f.